.claude/skills/graphify/SKILL.md

```
@@ -0,0 +1,137 @@
+# macOS managed envs: use pipx install graphifyy instead
+
+# Install the /graphify slash command into Claude Code
+graphify install
+
+# Build the initial graph for this repo
+cd /path/to/local-llm-server
+graphify .
+```
+
+On success you'll have:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)